tools/app-provisioning-tool/tests/Tests.csproj

```diff
@@ -1,6 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
+    // Test says that app-provisioning-lib is not compatible with net7.0!
```

Comment:
Can you clarify? What is the action item here?

Reply:
Ah thank you for catching that! We were running into an issue with building the project previously where I did not have the correct install -- that is resolved now.